### PR TITLE
Arista 7280CR3 Family

### PR DESCRIPTION
**Pull request by suom1**
- Updated PSU information for `DCS-7280CR3-32P4` 
- Fixed incorrect port layout for `DCS-7280CR3-32D4`
- Updated PSU information for `DCS-7280CR3-32D4`
- Adding new devices (`DCS-7280CR3K-32D4`, `DCS-7280CR3K-32P4`, `DCS-7280CR3K-32D4A`, `DCS-7280CR3K-32P4A`, `DCS-7280CR3-96`, `DCS-7280CR3K-96`)
- Adding new PSU modules

### CODE REVIEW

File: device-types/Arista/DCS-7280CR3-96.yaml [hunk 1]
```diff
@@ -0,0 +1,216 @@
+---
+manufacturer: Arista
+model: DCS-7280CR3-96
+slug: arista-dcs-7280cr3-96
+part_number: DCS-7280CR3-96
+u_height: 2
+is_full_depth: true
```
Comment (danner26) [inline device-types/Arista/DCS-7280CR3-96.yaml:7]: Please add the `airflow` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-32D4.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+manufacturer: Arista
+model: DCS-7280CR3K-32D4
+slug: arista-dcs-7280cr3k-32d4
+part_number: DCS-7280CR3K-32D4
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-32D4.yaml:7]: Please add the `airflow` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-32D4A.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+manufacturer: Arista
+model: DCS-7280CR3K-32D4A
+slug: arista-dcs-7280cr3k-32d4a
+part_number: DCS-7280CR3K-32D4A
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-32D4A.yaml:7]: Please add the `airflow` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-32P4.yaml [hunk 4]
```diff
@@ -0,0 +1,96 @@
+---
+manufacturer: Arista
+model: DCS-7280CR3K-32P4
+slug: arista-dcs-7280cr3k-32p4
+part_number: DCS-7280CR3K-32P4
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-32P4.yaml:7]: Please add the `airflow` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-32P4A.yaml [hunk 5]
```diff
@@ -0,0 +1,96 @@
+---
+manufacturer: Arista
+model: DCS-7280CR3K-32P4A
+slug: arista-dcs-7280cr3k-32p4a
+part_number: DCS-7280CR3K-32P4A
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-32P4A.yaml:7]: Please add the `airflow` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-96.yaml [hunk 6]
```diff
@@ -0,0 +1,216 @@
+---
+manufacturer: Arista
+model: DCS-7280CR3K-96
+slug: arista-dcs-7280cr3k-96
+part_number: DCS-7280CR3K-96
+u_height: 2
+is_full_depth: true
```
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-96.yaml:7]: Please add the `airflow` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-96.yaml:7]: Please add the `weight` and `weight_unit` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-32P4A.yaml (hunk 5 above)
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-32P4A.yaml:7]: Please add the `weight` and `weight_unit` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-32P4.yaml (hunk 4 above)
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-32P4.yaml:7]: Please add the `weight` and `weight_unit` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-32D4A.yaml (hunk 3 above)
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-32D4A.yaml:7]: Please add the `weight` and `weight_unit` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3K-32D4.yaml (hunk 2 above)
Comment (danner26) [inline device-types/Arista/DCS-7280CR3K-32D4.yaml:7]: Please add the `weight` and `weight_unit` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

File: device-types/Arista/DCS-7280CR3-96.yaml (hunk 1 above)
Comment (danner26) [inline device-types/Arista/DCS-7280CR3-96.yaml:7]: Please add the `weight` and `weight_unit` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Arista/DCS-7280CR3-96.yaml (hunk 1 above)
Comment (suom1) [reply device-types/Arista/DCS-7280CR3-96.yaml:9]: @danner26 Arista has 3 different SKU's (the one mostly used, all but one in this repo is without airflow).
Do you want me to change that in this PR to use the airflow SKU's instead which ends with -R or -F

**Comment by danner26** [comment]
Hello, I have updated the PR to inslude both new types. Once that is merged we can proceed with this.

**Pull request merged**